site/docs/hosting/heroku.md

```diff
@@ -178,7 +178,7 @@ We have now finished coding our main files.
 But before we go to the deployment steps, we can optimize our bot a little bit.
 As usual, this is optional.
 
-::: tip ⚡ Optimization (optional)
+::: tip ⚡ Optimization (Optional)
```

Comment:
For consistency with heading style, and as per https://github.com/grammyjs/website/pull/30#discussion_r735909521
```suggestion
::: tip ⚡ Optimization (optional)
```